<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>461</width>
     <height>491</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../qt/1x/Fichier 5.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>501</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../qt/1x/Fichier 2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(151, 249, 227);
border-radius:16px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="gestionReservations">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(250, 224, 176);
font: 16pt &quot;Book Antiqua&quot;;
border-radius:20px;
}

QPushButton:hover{
	background-color: rgb(248, 215, 148);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>      Gestion Résérvations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GestionPersonnel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(250, 224, 176);
font: 16pt &quot;Book Antiqua&quot;;
border-radius:20px;
}

QPushButton:hover{
	background-color: rgb(248, 215, 148);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>  Gestion Personnel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GestionEvenement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(250, 224, 176);
font: 16pt &quot;Book Antiqua&quot;;
border-radius:20px;
}

QPushButton:hover{
	background-color: rgb(248, 215, 148);
	color: rgb(255, 255, 255);
}

</string>
    </property>
    <property name="text">
     <string>      Gestion Evènements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gestionMateriels">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(250, 224, 176);
font: 16pt &quot;Book Antiqua&quot;;
border-radius:20px;
}

QPushButton:hover{
	background-color: rgb(248, 215, 148);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string> Gestion Matériels</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Book Antiqua&quot;;
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>SERVICES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel:hover{
	
	color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/public-service.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/public-service1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gestionClients">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(250, 224, 176);
font: 16pt &quot;Book Antiqua&quot;;
border-radius:20px;
}

QPushButton:hover{
	background-color: rgb(248, 215, 148);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>      Gestion Clients</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/311049087_853191362701555_96167479992419810_n.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Lucida Handwriting&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>TALENT HUB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/ajout-dutilisateur.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/calendrier-verification.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/portrait.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/calendrier-lignes-stylo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/lordinateur.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Book Antiqua&quot;;
color: rgb(81, 146, 81);</string>
   </property>
   <property name="text">
    <string>Votre meilleure solution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>120</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Book Antiqua&quot;;
color: rgb(81, 146, 81);</string>
   </property>
   <property name="text">
    <string>Pour des services meilleurs !</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_4</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_14</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
